client/signup/jetpack-connect/new-site/jetpack-logo.jsx

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Why is this component here, could it be moved to `client/components`? Also, would be nice to remove the svg from `/public/images/jetpack/jetpack-logo.svg`.